platformio.ini

```diff
@@ -110,8 +111,8 @@ lib_deps =
   h2zero/NimBLE-Arduino@1.4.0
   arduino-libraries/NTPClient@^3.1.0
   lorol/LittleFS_esp32@^1.0.6
-  https://github.com/meshtastic/AXP202X_Library.git#8404abb6d4b486748636bc6ad72d2a47baaf5460
-
+  https://github.com/lewisxhe/XPowersLib.git    
```

Comment:
Does this lib have an upstream somewhere? We'd like to prioritize upstream libs. Also our older AXP202X lib does have some local mods, are these incorporated in the new lib? E.G. is this a drop in replacement for our custom version?

Reply:
XPowersLib is specially written by me for AXP2101. In order to be compatible with the old TBeam, it also added support for AXP192 and improved some bugs in AXP20X_Library. I have registered XPowersLib on platfromio, and I will use the marked version later.

I know that the meshtastic AXP20X_Library has been modified locally, so XPowersLib has also added `class HasBatteryLevel` to be compatible with Meshtastic

https://github.com/lewisxhe/XPowersLib/blob/a0b581a5dd9c03efb7cd767236a2a421bc8f797e/src/XPowersLibInterface.hpp#L108